ansys/mapdl/core/launcher.py

```diff
@@ -145,9 +145,28 @@ def port_in_use(port, host=LOCALHOST):
         except:
             return True
 
+def delete_ip_file(path):
+    """Delete the created 'mylocal.ip'."""
+    # path=''
+    file_name = os.path.join(path,'mylocal.ip') 
+    if os.path.exists(file_name):
+        # os.remove(file_name)
+        pass 
 
+def create_ip_file(ip, path):
+    """Create 'mylocal.ip' file required for ansys to change the IP of the gRPC server."""
+    delete_ip_file(path) # Delete if exist
```

Comment:
Probably doesn't need to be it's own function, include this within `create_ip_file`.

Reply:
Nevermind, but you don't need to delete the file if it already exists, merely use the write mode (`"w"`) when opening the file.

Reply:
In the new commit I use the write mode (`"w"`).